cli/src/plugins/javadoc/index.ts

```diff
@@ -182,6 +183,57 @@ function makeSuperclassList(project: Project, doc: ParsedClassDoc) {
   ];
 }
 
+function makeInterfaceMethodList(project: Project, doc: ParsedClassDoc, interfaceTypes: Type[], inheritedMethods: {[k: string]: string[]}) {
```

Comment:
I would avoid making functions with long positional arguments lists. Instead, make a type called like `MakeInterfaceMethodListArgs` that has these members.

`inheritedMethods` could be `Record<string, string[]>` and could be aliased with a comment to explain what it is. (i.e. class qualified name -> array of method names).